src/test/common/steps/cookie-steps.js

```diff
@@ -1,10 +1,18 @@
-const { Then } = require('cucumber')
-const { expect } = require('chai')
+const { When, Then } = require('cucumber')
+const { assert, expect } = require('chai')
 
 const pages = require('./pages')
 
 const GOV_UK_TABE_CLASSNAME = 'govuk-table'
 
+When(/^I go directly to the cookies page$/, async function () {
+  await pages.cookies.open(pages.url)
+})
+
+When(/^I click on the cookies link$/, async function () {
+  await pages.enterName.clickCookieLink()
```

Comment:
You can access this directly from `genericPage`, e.g. `pages.genericPage.clickCookieLink()`.

Reply:
Updated